Doc/c-api/monitoring.rst

```diff
@@ -190,3 +190,9 @@ would typically correspond to a python function.
 .. c:function:: int PyMonitoring_ExitScope(void)
 
    Exit the last scope that was entered with :c:func:`!PyMonitoring_EnterScope`.
+
+
+.. c:function:: int PY_MONITORING_IS_INSTRUMENTED_EVENT(uint8_t ev)
+
+   Return true if the event corresponding to the event ID *ev* is
+   a :ref:`local event <monitoring-event-local>`.
```

Comment:
This matches the code.
@iritkatriel, does it match the intention for this API?

Reply:
It's been released in 3.13, so it's public now.

Reply:
Yes, we'll have to deprecate it. I doubt anyone is using it.

Reply:
Should I ask the SC if we can remove it without deprecation?

Reply:
The [approved](https://peps.python.org/pep-0387/#soft-deprecation) [term](https://docs.python.org/3/glossary.html#term-soft-deprecated) for a doc-only deprecation without a removal plan is “soft deprecation”.

```suggestion


.. c:function:: int PY_MONITORING_IS_INSTRUMENTED_EVENT(uint8_t ev)

   Return true if the event corresponding to the event ID *ev* is
   a :ref:`local event <monitoring-event-local>`.

   .. versionadded:: 3.13

   .. deprecated:: next

      This function is :term:`soft deprecated`.
```